css/main.css

```diff
@@ -458,6 +461,105 @@ body {
   margin: 18px 0;
 }
 
+.modal {
+  background: #fff;
+  padding: 1.5rem;
+  width: 90%;
+  opacity: 0;
+  visibility: hidden;
+  transition: transform 1s, opacity 1s, visibility 0s;
+  z-index: 300;
+}
+
+.modal__header {
+  justify-content: space-between;
+}
+
+.modal__header img {
+  width: 20px;
+  height: 20px;
+  cursor: pointer;
+}
+
+.modal__image {
+  width: 100%;
+  margin: 1rem 0;
+}
+
+.modal__dsc {
+  flex-direction: column;
+}
+
+.modal__dsc__first {
+  margin: 0.7rem 0;
+}
+
+.modal__dsc__second {
+  flex-direction: column;
+  align-items: flex-start;
+}
+
+.modal__dsc__second div {
+  margin: 1rem 0;
+}
+
+.modal__dsc__second p {
+  padding: 4px 12px;
+  border-radius: 8px;
+  margin-right: 0.5rem;
+}
+
+.modal__btn {
+  margin: 1rem 0;
+}
+
+.modal__btn li a {
+  border: 1px solid #6070ff;
+  padding: 0.6rem 1.5vw;
+  border-radius: 8px;
+  margin-right: 0.5rem;
+}
+
+.modal__feature ul {
+  padding: 0;
+  margin: 0;
+}
+
+.modal__feature ul li {
+  margin-left: 1.5rem;
+}
+
+.modal.active {
+  visibility: visible;
+  transform: scale(1);
+  opacity: 1;
+}
```

Comment:
- [ ] Nice work so far, in your desktop view the modal is not fitted to the screen making it difficult for the user to see the close button. kindly fix
see image below 👇 
    
![image](https://user-images.githubusercontent.com/66097371/162416605-dbac9691-386d-48f6-b7c1-f63b4bd074b6.png)

Reply:
Have effected the change. Will proceed to submit for approval now. Thanks a lot! @VICTOR-CHUKWU 
Please can you kindly check and confirm it's ok now. This is my last submission chance please. Thanks.